schema.sql

```diff
@@ -6,5 +6,6 @@ CREATE TABLE animals (
   date_of_birth date,
   escape_attempts int,
   neutered boolean,
-  weight_kg float
+  weight_kg float,
+  species varchar
 );
```

Comment:
- [ ] You are supposed to add species column with a different query but here you have added the species column in the previous query which is not what you are supposed to do. 😨 You can see below to get an idea what i am talking about 👍

```
CREATE TABLE animals (
  id int GENERATED BY DEFAULT AS IDENTITY,
  name varchar,
  date_of_birth date,
  escape_attempts int,
  neutered boolean,
  weight_kg float
);

   ALTER TABLE animals ADD species varchar(255);
```